hostlink/HostLink.cpp

```diff
@@ -404,6 +412,9 @@ void HostLink::boot(const char* codeFilename, const char* dataFilename)
 
   // Request to boot loader
   BootReq req;
+  static_assert(std::is_pod<BootReq>::value);
```

Comment:
@m8pple I probably wouldn't bother with this assert (I had to lookup `is_pod` and apparently it is being deprecated), but I'm ok to include it if you think its worth it.

Reply:
Yeah, it probably is a bit redundant. Even if it is non-POD, someone is still sending it down
a pipe as raw bytes, so presumably know what they are doing. I'll take it out.